docs/guides/deploy-warp-route.mdx

```diff
@@ -35,6 +35,11 @@ hyperlane config create warp
 If your config looks correct, you can now skip to [Step 2: Deployment](#2-deployment).
 Or see below for details on how to define your config manually.
 
+If you want to use [custom chains](../deploy-hyperlane.mdx) in your Warp Route config, use the `--chains` flag. 
```

Comment:
instead of just linking, would be good to explain that to use custom chains you should first [deploy hyperlane] yourself before proceeding

ditto below

Reply:
Made changes @paulbalaji 
Plz have a check